lib/src/client.dart

```diff
@@ -4,7 +4,15 @@ import 'package:appstore_connect/src/model/model.dart';
 import 'package:appstore_connect/src/token.dart';
 import 'package:http/http.dart';
 
-const _apiUri = 'https://api.appstoreconnect.apple.com/v1/';
+extension AppStoreConnectUri on Uri {
+  static Uri v1({String? resource = ''}) {
+    return Uri.parse('https://api.appstoreconnect.apple.com/v1/$resource');
+  }
+
+  static Uri v2({String? resource = ''}) {
+    return Uri.parse('https://api.appstoreconnect.apple.com/v2/$resource');
+  }
+}
```

Comment:
[Anmerkung] Da diese beide Methoden nur zusätzliche statische Methoden bereitstellen, würde ich sie nicht als einen Extension von Uri modellieren, da sie ja eigentlich gar nicht extenden. Vielleicht eher in einer abstrakten Klassen den wir dann als Namespace missbrauchen.

`resource` würde ich dann nicht als optionalen benannten Parameter sondern als optionalen positionalen Parameter abbilden. Das ist bei vielen Dart Methoden/Klassen so, die nur einen Parameter annehmen und nur eine Funktion haben. Dann spart sich das angeben von Resource.

Reply:
Done.
Ich hatte die Parameter erst als optionale positional Parameter. Leider muss man dann beim Initialisieren ohne Resource 'null' übergeben. Das fand ich damals nicht so schön. Kannst es dir mal [anschauen](https://github.com/programmierbar/appstore_connect/commit/bb7da3f46f2f33da112f747aa4916771106c1b48).